content/en/admin/scaling.md

```diff
@@ -412,6 +412,8 @@ REPLICA_DB_PORT
 REPLICA_DB_NAME
 REPLICA_DB_USER
 REPLICA_DB_PASS
+REPLICA_PREPARED_STATEMENTS (defaults to true)
```

Comment:
In the related PR, it actually defaults to the same value as `PREPARED_STATEMENTS`, not to `true`: if `PREPARED_STATEMTNTS` is set to anything but `true`, so will `REPLICA_PREPARED_STATEMENTS` by default.

Reply:
Expanded documentation. Please review.